src/transformStateWithClones.js

```diff
@@ -66,31 +66,24 @@
  */
 function transformStateWithClones(state, transforms) {
   const resultingArray = [];
```

Comment:
No need to include 'Array' in the name of your variable. 'clonedStates' would describe this variable better.